addons/hr_timesheet/views/project_views.xml

```diff
@@ -212,12 +212,14 @@
                         </form>
                     </field>
                     <group attrs="{'invisible': [('analytic_account_active', '=', False)]}">
-                        <group class="oe_subtotal_footer oe_right" name="project_hours">
-                            <span class="o_td_label float-start">
-                                <label class="fw-bold" for="effective_hours" string="Hours Spent" attrs="{'invisible': [('encode_uom_in_days', '=', True)]}"/>
-                                <label class="fw-bold" for="effective_hours" string="Days Spent" attrs="{'invisible': [('encode_uom_in_days', '=', False)]}"/>
-                            </span>
-                            <field name="effective_hours" widget="timesheet_uom" nolabel="1"/>
+                        <group class="oe_subtotal_footer d-block" name="project_hours">
+                            <div class="d-flex" colspan="2">
```

Comment:
In stable, we cannot change the xml structure because a custom module could do a xpath and select one xml element in that document. 😬